src/components/FormInput.js

```diff
@@ -10,6 +10,33 @@ function FormInput({ id, name, value, onChange }) {
     setIsShown(!isShown);
   }
 
+  const autoExpand = function (textField) {
+    // Reset field height
+    textField.style.height = "inherit";
+
+    // Get the computed styles for the element
+    const computed = window.getComputedStyle(textField);
+
+    // Calculate the height
+    const height =
+      parseInt(computed.getPropertyValue("border-top-width")) +
+      parseInt(computed.getPropertyValue("padding-top")) +
+      textField.scrollHeight +
+      parseInt(computed.getPropertyValue("padding-bottom")) +
+      parseInt(computed.getPropertyValue("border-bottom-width"));
+
+    textField.style.height = height + "px";
+  };
+
+  document.addEventListener(
```

Comment:
You could add this an event listener just to the `textarea` in the regular react way `<textarea onInput={handleInput}`.
Also, remember that when adding events listener to the document or window (or not handled in the jsx of the component) we should use `useEffect` and make sure we remove the event listener in the cleanup function.